input/core/src/commonMain/kotlin/majestic/editor/ToolBarTabColors.kt

```diff
@@ -0,0 +1,8 @@
+package majestic.editor
+
+import androidx.compose.ui.graphics.Color
+
+data class StateColors(
```

Comment:
Because this resides in majestic, we should its safe to give this class a much more descriptive name (.e.g `ToolButtonColor`)